### PR TITLE
LPS-183059 Add AI-Creator WebContent configuration using the auto-generated view

### PR DESCRIPTION
**Pull request by lfbesada**
https://issues.liferay.com/browse/LPS-183059

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:relevant

File: modules/apps/ai-creator/app.bnd [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Liferay-Releng-App-Description:
+Liferay-Releng-App-Title: ${liferay.releng.app.title.prefix} Ai Creator
```
Comment (ealonso) [inline modules/apps/ai-creator/app.bnd:2]: @lfbesada Shouldn't be it Ai in uppercase?

Comment (lfbesada) [reply modules/apps/ai-creator/app.bnd:2]: true! thanks for catching it!

Comment (lfbesada) [reply modules/apps/ai-creator/app.bnd]: @ealonso Done!

File: modules/apps/ai-creator/ai-creator-open-ai-impl/scr/main/java/com/liferay/ai/creator/open/ai/internal/configuration/manager/AICreatorOpenAIConfigurationManagerImpl.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
```
Comment (ealonso) [inline modules/apps/ai-creator/ai-creator-open-ai-impl/scr/main/java/com/liferay/ai/creator/open/ai/internal/configuration/manager/AICreatorOpenAIConfigurationManagerImpl.java:2]: @lfbesada you put it under scr folder

Comment (lfbesada) [reply modules/apps/ai-creator/ai-creator-open-ai-impl/scr/main/java/com/liferay/ai/creator/open/ai/internal/configuration/manager/AICreatorOpenAIConfigurationManagerImpl.java]: Done! thanks for catching it! 🤦

**Comment by lfbesada** [comment]
see https://github.com/liferay-echo/liferay-portal/pull/12032

**Pull request closed without merging**